### PR TITLE
README 2.0 API updates

### PR DESCRIPTION
**Pull request by BillyAbildgaard**
updated README to show you can use 2.0 API and updated how to get active graph on a node using 2.0 API

@RackHD/corecommitters @keedya @srinia6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,6 +186,13 @@ GET
 /api/1.1/nodes/<id>/workflows/active
 ```
 
+**2.0 API to list active workflows running against the node
```
Comment (heckj) [inline README.md]: singular is better - don't think we can run more than one against a node:

`**2.0 API to list the active workflow running against the node`

**Comment by srinia6** [comment]
+1

**Comment by BillyAbildgaard** [comment]
@heckj good catch, change made

**Comment by heckj** [comment]
👍

**Comment by anhou** [comment]
:+1:

**Comment by BillyAbildgaard** [comment]
Dear Jenkins,

Test this please.

**Pull request merged**